R/utils_track_progress.R

```diff
@@ -159,32 +162,33 @@ writeNarrativeDataFrame <- function(tadat) {
   }
 
 
+  
   # Flagging Tab
   for (flag in tadat$selected_flags) {
     df[nrow(df) + 1, ] <- c("Selected Flag", flag)
   }
-
-
+  
+  
   if (!is.null(tadat$m2f)) {
     df[nrow(df) + 1, ] <- c("Depth unit conversion", tadat$m2f)
   } else {
     df[nrow(df) + 1, ] <- c("Depth unit conversion", "None")
   }
 
-
   # Filtering tab
-  for (row in 1:nrow(tadat$selected_filters)) {
-    df[nrow(df) + 1, ] <- c(
-      "Selected Filter",
-      paste0(
+  if (nrow(tadat$selected_filters) > 0) {
```

Comment:
Is this edit fixing a different issue you ran into?

Reply:
This was one of the first things I tried to resolve the zipfile issue. It didn't fix it, but I think it's a good idea to keep it. This just skips the recording of selected filters in the progress file if the filters haven't been selected yet.